src/services/ai-server/ai-persona-service/dto/external.config.ts

```diff
@@ -1,6 +1,44 @@
-import { Field, InputType } from '@nestjs/graphql';
+import {
+  Field,
+  InputType,
+  ObjectType,
+  registerEnumType,
+} from '@nestjs/graphql';
 
-@InputType('ExternalConfig')
+enum OpenAIModel {
```

Comment:
that's OK for this version but it may be worth considering either putting it in the config or in the domain model, as we'd like some filtering, and these models will change (and we don't want the code to break / or have mandatory hotfix when the models change).